Cargo.toml

```diff
@@ -38,7 +38,7 @@ walkdir = { version = "2.3.2", optional = true }
 # NOTE: we don't depend on this crate but we need to activate this feature otherwise it's super slow
 walrus = { version = "0.19.0", features = ["parallel"] }
 wasm-bindgen-cli-support = "0.2.68"
-xtask-watch = { version = "0.1.5" }
```

Comment:
it's okay that you do it but it is unnecessary because dependencies are always selected with the highest minor and patch version.

so when you write xtask-watch = 0.1.0 you will actually pull the 0.1.6 version anyway

this is the reason why you need to bump the major when you break the api